config/zones.json

```diff
@@ -3708,6 +3708,31 @@
     },
     "timezone": null
   },
+  "US-SEC": {
+    "_comment": "https://www.seminole-electric.com/facilities/generation/ lists two 650MW coal-fired plants, one 810MW gas plant, and a small 2.2MW solar farm. Hwoever, EIA combines coal and gas generation data, so we report those together as unknown",
```

Comment:
can you put this comment in the parser itself? This is the best place to put this IMO

Reply:
good call! done.

Reply:
great! Then I think you can remove it from here.